packages/python/src/components/PyDoc.tsx

```diff
@@ -1,10 +1,10 @@
 import { For, Indent, List, Prose, Show, childrenArray } from "@alloy-js/core";
 import { Children } from "@alloy-js/core/jsx-runtime";
 import { ParameterDescriptor } from "../parameter-descriptor.js";
-import { Atom } from "./index.js";
+import { Atom, TypeProps } from "./index.js";
```

Comment:
Should we be importing `type TypeProps`?

Reply:
For some reason this wasn't raising an error, but I renamed the class to the correct one.